squire_web/src/tournament/rounds/selected.rs

```diff
@@ -4,6 +4,17 @@ use squire_sdk::model::{
 };
 use yew::prelude::*;
 
+pub fn round_info_display(rnd: &Round) -> Html {
```

Comment:
This works for now, but this function and the player version of this function should probably be moved to a more common location, probably in `src/tournament/views.rs` (new file), `src/tournament/utils.rs` (new file), or `src/tournament/mod.rs`.

Eventually, these should probably be their own types. There is only one between how we want to display `SelectedRound` and a player's info selected round. For the `SelectedRound`, we want to allow users to be able to select a player from the list of players in the round. For the player's info selected round, clicking a user will either do nothing or update the `SelectedPlayer`. We should be able to manage this with just a change to a `Callback` field.